src/core_plugins/timelion/public/directives/timelion_expression_input_helpers.js

```diff
@@ -73,6 +73,26 @@ function inLocation(cursorPosition, location) {
   return cursorPosition >= location.min && cursorPosition <= location.max;
 }
 
+function getArgumentsHelp(functionHelp, functionArgs = []) {
+  if (!functionHelp) {
+    return [];
+  }
+
+  // Do not provide 'inputSeries' as argument suggestion for chainable functions
+  const argsHelp = functionHelp.chainable ? functionHelp.args.slice(1) : functionHelp.args.slice(0);
+
+  // ignore arguments that are all ready provided in function declaration
```

Comment:
Typo "all ready" -> "already"

Reply:
And just so I understand, this means we're excluding arguments that the user has already added to the function call, right?

Reply:
that is correct.